oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -201,6 +201,18 @@ storage:
     batchEnabled: ${SW_STORAGE_INFLUXDB_BATCH_ENABLED:true}
     fetchTaskLogMaxSize: ${SW_STORAGE_INFLUXDB_FETCH_TASK_LOG_MAX_SIZE:5000} # the max number of fetch task log in a request
     connectionResponseFormat: ${SW_STORAGE_INFLUXDB_CONNECTION_RESPONSE_FORMAT:MSGPACK} # the response format of connection to influxDB, cannot be anything but MSGPACK or JSON.
+  postgresql:
```

Comment:
1. Storage documentation is required to update
2. `configuration-vocabulary.md` is required to update
3. `known-oap-backend-dependencies.txt` and `known-oap-backend-dependencies-es7.txt` are required to updated, your CI will fail because of this.
4. LICENSE in the `dist-material/dist-material` is required to update, and which license is postgresql using?

Reply:
What does `Storage documentation is required to update` mean.

Reply:
The document you read every day, where do you learn how to set up SkyWalking? `backend-storage.md`.